src/server/graphql/mutations/helpers/extendNewMeetingForType.js

```diff
@@ -1,14 +1,11 @@
 import {RETROSPECTIVE} from 'universal/utils/constants';
 
 const extendNewMeetingForType = (newMeetingBase) => {
-  if (newMeetingBase.meetingType === RETROSPECTIVE) {
-    return {
-      ...newMeetingBase,
-      reflectionGroups: [],
-      reflections: []
-    };
+  switch (newMeetingBase.meetingType) {
+    case RETROSPECTIVE:
```

Comment:
it'd be nice to be guaranteed an array here in our backend, otherwise someone could end a meeting without creating groups & then we'd have to do safety checks to make sure groups is an array. that gets annoying.